README.md

```diff
@@ -55,6 +56,31 @@ You also may need to change the source URIs for components. We recognize this is
 
 If you have your own mirror of the components of puppet-agent, you can also use a rewrite rule. See the [Vanagon README](https://github.com/puppetlabs/vanagon/blob/master/examples/projects/project.rb#L26) for an example.
 
+
+Building puppet-agent for windows
```

Comment:
No, we will not be going back to pftw, so yes we will need to put this PR back in.